development/check-scripts.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+#
+# This is a helpers script for validating bash scripts inside the test-infra repository.
```

Comment:
MINOR:
now we have 2 scripts in development dir:
- check-scripts.sh
- check.sh

maybe we should rename this one to validate-scripts.sh